docs/Getting-Started.md

```diff
@@ -0,0 +1,38 @@
+# Getting Started
+
+## Deployment
+
+You can deploy ShinyCMS to Heroku using the included Procfile, after setting
```

Comment:
Perhaps a candidate for a Deploy To Heroku button? https://devcenter.heroku.com/articles/heroku-button